### PR TITLE
Pva/rdfbrowser 493 update libraries frontend

### PR DESCRIPTION
**Pull request by peter-va**
updated libraries to resolve high vulnerabilities in javascript and java code

replaced xlsx with xlsx-republish (redistribution via xlsx of xlsx after weird drama caused removal from npm)

also apparently History is a reserved word in excel so i renamed the history tab in the concept details export to Concept History

### CODE REVIEW

File: web/build.gradle [hunk 1]
```diff
@@ -67,11 +67,18 @@ dependencies {
 
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.cloud:spring-cloud-starter-gateway-mvc'
+    implementation 'ch.qos.logback:logback-classic:1.4.12' // Updated version
```
Comment (gaaliciA1990) [inline web/build.gradle:70]: In my IDE I'm seeing this has the following vulnerability
```
CVE-2023-6481 7.5 Deserialization of Untrusted Data vulnerability with High severity found
```
Not sure if this is something we are okay with or not. Can you confirm?

File: web/frontend/src/app/component/concept-display/concept-display.component.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { ConceptDetailService } from './../../service/concept-detail.service';
 import { Concept } from './../../model/concept';
 import { ConfigurationService } from '../../service/configuration.service';
 import { Subject } from 'rxjs';
-import { writeXLSX, utils, WorkSheet } from 'xlsx';
+import { writeXLSX, utils, WorkSheet } from 'xlsx-republish';
```
Comment (gaaliciA1990) [inline web/frontend/src/app/component/concept-display/concept-display.component.ts:8]: I tried running the frontend locally, npm start, and I'm getting an error with this import causing the system to fail compile
```
./src/app/component/concept-display/concept-display.component.ts:12:0-50 - Error: Module not found: Error: Can't resolve 'xlsx-republish' in '/Users/gaalicia/Documents/nci/evs-explore/web/frontend/src/app/component/concept-display'

Error: src/app/component/concept-display/concept-display.component.ts:8:45 - error TS2307: Cannot find module 'xlsx-republish' or its corresponding type declarations.

8 import { writeXLSX, utils, WorkSheet } from 'xlsx-republish';
 ```

**Review by gaaliciA1990 — COMMENTED** [review]
I see this is working on dev fine, but I was not able to start this locally. Maybe I need to set up something? Just want to confirm this and a few comments below.

File: web/build.gradle (hunk 1 above)
Comment (peter-va) [reply web/build.gradle:70]: Interesting. We want to clear everything critical or high so I'll need to take another look at that.

Comment (peter-va) [reply web/build.gradle:70]: `npm i` or `npm install xlsx-republish` should fix that

File: web/frontend/src/app/component/concept-display/concept-display.component.ts (hunk 2 above)
Comment (peter-va) [reply web/frontend/src/app/component/concept-display/concept-display.component.ts:8]: `npm i` or `npm install xlsx-republish` should fix that

**Review by gaaliciA1990 — APPROVED** [review]
The system builds and runs as expected and no issues found. Ready for mergining

**Pull request merged**